3604/1/gem_info_fetcher.rb

```diff
@@ -0,0 +1,81 @@
+require 'mechanize'
+
+# class for get github url something gem and get information about this gem
+class GemInfoFetcher
+  URL = 'https://rubygems.org/'.freeze
+  SHOT_URL_USED_BY = '/network/dependents'.freeze
+  USED_BY_CSS_CLASS = "a[class='btn-link selected']".freeze
+  CONTRIBUTORS_CSS_CLASS = "span[class='num text-emphasized']".freeze
+  WATCH_STAR_FOEK_CSS_CLASS = '.social-count'.freeze
+
+  def initialize(name_gem)
+    @name_gem = name_gem
+  end
+
+  def information_about_gem
+    hash_with_info_about_gem if such_gem?
+  end
+
+  private
+
+  def such_gem?
+    begin
+      page_gems
+    rescue Mechanize::ResponseCodeError
+      puts "No such gem #{@name_gem}"
+      return false
+    end
+    github_url
```

Comment:
do you need this line? `such_gem?` is used just to check that gem exist.
also maybe `gem_exists?` is better naming here?